cg/constants/constants.py

```diff
@@ -162,10 +162,12 @@ class FileFormat(StrEnum):
 
 
 class GenomeVersion(StrEnum):
+    GRCh37: str = "GRCh37"
+    GRCh38: str = "GRCh38"
+    T2T_CHM13: str = "T2T-CHM13v2.0"
+    canfam3: str = "canfam3"
     hg19: str = "hg19"
     hg38: str = "hg38"
```

Comment:
probably out of scope, but why are these Enums lower case?

Reply:
no idea, I've changed corrected them